### PR TITLE
west: update manifest to the new zscilib revision.

### PR DESCRIPTION
**Pull request by uLipe**
Signed-off-by: Felipe Neves <felipe.neves@linaro.org>

### CODE REVIEW

**Comment by uLipe** [comment]
@nashif any hints reagarding the flaky behavior of the compliance check? it sometimes fails when update the zsl to the desired revision in manifest file.

**Comment by microbuilder** [comment]
> @nashif any hints reagarding the flaky behavior of the compliance check? it sometimes fails when update the zsl to the desired revision in manifest file.

@uLipe The revision here is incorrect. The PR hasn't been merged yet, but you are adding the commit hash directly. Until the PR is merged you should have the following in the manfest:

```yaml
revision: pull/45/head
```

File: west.yml [hunk 1]
```diff
@@ -257,7 +257,7 @@ manifest:
       path: modules/lib/zcbor
     - name: zscilib
       path: modules/lib/zscilib
-      revision: ca070ddabdaf67175a2da901d0bd62e8899371c5
+      revision: 075878f002f997d515c99467148e621f6c0d1680
```
Comment (microbuilder) [inline west.yml:260]:
```suggestion
      revision: pull/45/head
```

**Review by microbuilder — CHANGES_REQUESTED** [review]

**Comment by uLipe** [comment]
@gmarull PTAL

File: west.yml [hunk 2]
```diff
@@ -257,7 +257,7 @@ manifest:
       path: modules/lib/zcbor
     - name: zscilib
       path: modules/lib/zscilib
-      revision: ca070ddabdaf67175a2da901d0bd62e8899371c5
+      revision: pull/45/head
```
Comment (microbuilder) [inline west.yml:260]: This needs to be changed to the commit hash now that the other PR has been merged.

**Review by microbuilder — CHANGES_REQUESTED** [review]

File: west.yml (hunk 2 above)
Comment (uLipe) [reply west.yml:260]: It is already changed

**Review by gmarull — DISMISSED** [review]

**Review by microbuilder — DISMISSED** [review]

**Comment by uLipe** [comment]
Just rebased!

**Review by gmarull — APPROVED** [review]

**Review by microbuilder — APPROVED** [review]

**Pull request merged**